content/en/news/releases/1.17.x/announcing-1.17/change-notes/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Istio 1.17.0 Change Notes
+linktitle: 1.17.0
+subtitle: Minor Release
+description: Istio 1.17.0 change notes.
+publishdate: 2023-02-14
+release: 1.17.0
+weight: 10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** setting `PILOT_CERT_PROVIDER` to `kubernetes` for Kubernetes versions less than 1.20. [PR](https://github.com/istio/istio/pull/42233)
+
+- **Deprecated** Lightstep provider, please use OpenTelemetry provider instead. ([Issue #40027](https://github.com/istio/istio/issues/40027)) [PR](https://github.com/istio/api/pull/2663)
+
+## Traffic Management
+
+- **Improved** `MostSpecificHostMatch` to prevent full scanning hosts when encountering wildcards. [Issue #41453]( https://github.com/istio/istio/issues/41453) [PR](https://github.com/istio/istio/pull/41610)
+
+- **Improved** Gateway naming conventions to be concatenation of `Name` and `GatewayClassName`. Deployment also now deploys with its own Service Account, rather than using the `default` token. Naming convention affects name of Deployment, Service and Service Account. [PR](https://github.com/istio/istio/pull/43103)
```

Comment:
```suggestion
- **Improved** Gateway naming conventions to be the concatenation of `Name` and `GatewayClassName`. Deployment also now deploys with its own Service Account, rather than using the `default` token. Naming convention affects name of Deployment, Service and Service Account. [PR](https://github.com/istio/istio/pull/43103)
```

Reply:
Is this only for the Kubernetes gateway?

Reply:
I believe so.